### DISCUSSION TITLE
Not working anymore for me

### DISCUSSION THREAD

**Discussion by HSteindl** [DarwinsBuddy/WienerNetzeSmartmeter#48] [General]
Hello
Most likely I am not the only one, but mine stopped working around Jan 17/18th-ish. 
I wiped the custom_component directory and re-uploaded the latest code from here, but no matter what I try, the login user/password fails. I do not get past the authentication of the plugin...
Hmm, bummer
HST

**Comment by DarwinsBuddy** [comment]
Hi @HSteindl 
You are not the only one, though we released a new version a couple of days ago which should incorporate the changes necessary for the integration to talk to the WienerNetze API again.

I don't know which commit hash you're currently on but please consider setting up [HACS](https://hacs.xyz/)  on your Home Assistant, s.t. you get informed when there is another update from our (or potentially other community) projects to be installed.

I double checked the current version [v1.0.1](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/releases/tag/v1.0.1)
and login as well as device updates work as expected.

Have a nice day. :relaxed:

**Reply by HSteindl** [reply]
Hmm, looks like there is some caching going on. Simply erasing and re-uploading the new code does not cut it. After a reboot I am back online again. Thank you for your emotional support! ;-)
HST

**Reply by DarwinsBuddy** [reply]
Glad to hear that everything works. :heart: 
Though I can merely cordially invite you to try out HACS (if it fits your needs) to take care of the updates of your components. It's hassle-free and saves a lot of time and afford :)
